aineko/extras/connectors/rest_poller.py

```diff
@@ -0,0 +1,260 @@
+# Copyright 2023 Aineko Authors
+# SPDX-License-Identifier: Apache-2.0
+"""Extra module for connecting to a REST endpoint."""
```

Comment:
Maybe we should document "why" a user might want to use this node. I don't think it's mentioned anywhere what the use case is and how it might benefit users.

Reply:
Sure, but this seems better suited to showcase in the docs or in the docstring so it shows up in the docs.